bindings/ruby/lib/typelib/clang.rb

```diff
@@ -69,6 +69,20 @@ def self.load(registry, file, kind, options)
             end
         end
 
+        def self.clang_binary_name
+            if !`which clang-3.4`.empty?
+                return "clang-3.4"
+            elsif !`which clang`.empty?
+                IO.popen(['clang', '--version']) do |clang_io|
+                    clang_version = clang_io.read
+                    if clang_version.include?('clang version 3.4')
+                        return "clang"
+                    end
+                end
+            end
+            raise SystenCallError, "Couldn't find clang 3.4 compiler binary!"
```

Comment:
Don't use SystemCallError, this is not a system call. RuntimeError would be fine in this case.